snap/snapcraft.yaml

```diff
@@ -0,0 +1,37 @@
+name: sisyphus
+base: core18
+version: '0.1'
+summary: Multi-agent simulation tool
+description: |
+  Sisyphus is a multi-agent simulation model for testing web and messaging
+  systems. Agents are represented as state machines that can execute state
+  transitions and generate activity with statistical probability.
+
+grade: devel
+confinement: strict
+
+apps:
+  sisyphus:
+    command: sisyphus-wrapper.bash
+    plugs:
+    - network
+    - network-bind
+
+parts:
+  sisyphus:
+    plugin: go
+    source: .
+    go-packages:
+    - github.com/cloud-green/sisyphus/cmd/sisyphus
+    override-build: |-
```

Comment:
For some reason, the snapcraft go plugin doesn't work and I had to do this as a workaround :man_shrugging: